src/app/pages/installation/_linux_designer.twig

```diff
@@ -0,0 +1,30 @@
+<p>
+  1. Download the Axon Ivy Designer to your desired location
+<ul class="actions">
+  <li><a href="{{downloadUrl}}" class="button special"><i class="si si-download"></i> Download Axon Ivy Designer {{bugfixVersion}}</a></li>
+</ul>
+<img src="/images/installation/download-designer.png" />
+</p>
+
+<p>
+  2. Extract the downloaded zip file and navigate into the extract Axon.ivy Designer directory.
+  <img src="/images/installation/linuxDesigner/ubuntuExtract.png" />
+</p>
+
+
+<p>
+  3. Run the <code>install-dependencies.sh</code> script in a Terminal, in order to install crucial dependencies.
+  <img src="/images/installation/linuxDesigner/ubuntuRunInstallDeps.png" />
+  Supply your sudoer password in order to allow privileged installers to run:
+  <img src="/images/installation/linuxDesigner/ubuntuInstallDepsTerminal.png" />
+</p>
+
+<p>
+  4. If you work on Ubuntu 20.04 or newer, switch to X.org Window system by logging out, clicking on the settings button
```

Comment:
why is this needed?

Reply:
process-editor won't show any contents: maybe we can remove this once we have only the GLSP editor.
yet, there are also other minor issues on wayland, workspace selector is sometimes buggy.
![ubuntu2110-awtBridge](https://user-images.githubusercontent.com/15085693/143587185-1a2c5bcb-7c63-43fc-b2c7-b2ee336fd0c7.png)
![ubuntuDesktop21 10 on QEMU-KVM_001](https://user-images.githubusercontent.com/15085693/143587241-09cf4d54-016a-4b96-8a26-0d11c9503842.png)

Reply:
Thanks for your fast feedback @alexsuter